test/src/API/remote-bind.test.js

```diff
@@ -44,29 +34,59 @@ describe.skip('Remote Bind tests', () => {
 
     after(async function() {
         this.timeout(testTimeout.med);
-        await projectService.unbindProject(projectID);
-        await projectService.deleteProjectDir(remoteProjectName);
-        await projectService.deleteProjectDir(localProjectName);
+        await projectService.removeProject(pathToLocalRepo, projectID);
     });
 
-    describe('Complete remote-bind', () => {
-        describe('Success Case', () => {
-            it('succeeds in creating a project, uploading files to it, and then building the project', async function() {
-                this.timeout(testTimeout.med);
-                const res = await testRemoteBindStart(remoteProjectName);
-                projectID = res.body.projectID;
-                const projectIDs  = await projectService.getProjectIDs();
-                projectIDs.should.include(projectID);
-                await testRemoteBindUpload(projectID, remoteProjectName, pathToLocalRepo);
-                await testRemoteBindEnd(projectID);
+    describe('Complete bind (these `it` blocks depend on each other passing)', () => {
+        it('returns 202 when starting the bind process', async function() {
+            this.timeout(testTimeout.med);
+
+            const res = await projectService.bindStart({
+                name: projectName,
+                language: 'nodejs',
+                projectType: 'nodejs',
+                path: pathToLocalRepo,
+                creationTime: Date.now(),
+            });
+
+            res.should.have.status(202);
+            const expectedFields = ['projectID', 'name', 'workspace'];
+            expectedFields.forEach(field => {
+                res.body[field].should.not.be.null;
             });
+            projectID = res.body.projectID;
+        });
+        it('returns 200 for each file it uploads to PFE', async function() {
```

Comment:
`it('returns 200 for each file it uploads to PFE, and checks that a package.json file exists in PFE')`